### PR TITLE
Replace some prints by log messages

### PR DESCRIPTION
**Pull request by andywaltlova**
[HMS-3838](https://issues.redhat.com/browse/HMS-3838)
SSIA

- [x] Ideally needs log level from `rhc` https://github.com/oamg/rhc-worker-script/pull/119
- [x] Consider also logging to file

Resolves: https://github.com/oamg/convert2rhel-worker-scripts/issues/49

### CODE REVIEW

File: scripts/c2r_script.py [hunk 1]
```diff
@@ -8,6 +9,11 @@
 
 from urllib2 import urlopen, URLError
 
+# NOTE: What are all possible levels we can get from rhc?
```
Comment (andywaltlova) [inline scripts/c2r_script.py:12]: How about trace? :thinking:

Comment (r0x0d) [reply scripts/c2r_script.py]: We should stick with the levels from python, actually... I think what we can do is try to parse the level, if it is not a level that exists in the python mapping, we then default it to info.

File: scripts/c2r_script.py [hunk 2]
```diff
@@ -68,46 +87,53 @@ def _create(self, data):
...
             os.remove(self.path)
         except OSError as err:
-            print(err)
+            logger.warning(err)
             return False
         return True
 
     def restore(self):
         """Restores file backup (rename). Returns True if restored, otherwise False."""
+        file_path = os.path.join(self.path, self.backup_suffix)
```
[the remark this answers is not in the document]
Comment (r0x0d) [reply scripts/c2r_script.py:115]: Oh, that's true... Tried to be smart and just broke things :sweat: 
```suggestion
        file_path = self.path + self.backup_suffix
```

**Comment by andywaltlova** [comment]
Adjusted few tests here and there to increase coverage, now just some lines in logging functions are missing

**Review by r0x0d — APPROVED** [review]

**Pull request merged**